### DISCUSSION TITLE
How to merge cells in v-data-table like in Excel?

### DISCUSSION THREAD

**Discussion by gitercn** [vuetifyjs/vuetify#13770] [Q&A]
I'm using Vuetify for the current project. Now I need to create a table containing merged cells (including both columns and rows) that looks like the one in the Excel. It appears v-data-table does not have this feature built in. How to create it? (btw, some cells of the table should be able to allow user edit.)

![merge-cells](https://user-images.githubusercontent.com/53464166/120972287-2b4b9300-c7a0-11eb-8916-ae6a24dd8c9d.png)

**Comment by nekosaur** [comment]
You are correct that `v-data-table` does not have any internal support for col- or rowspan. You can use the `item` or `body` slot to render your own custom markup.
